openpype/hosts/maya/plugins/load/load_arnold_standin.py

```diff
@@ -0,0 +1,217 @@
+import os
+import clique
+
+import maya.cmds as cmds
+import mtoa.ui.arnoldmenu
+
+from openpype.settings import get_project_settings
+from openpype.pipeline import (
+    load,
+    get_representation_path
+)
+from openpype.hosts.maya.api.lib import (
+    unique_namespace, get_attribute_input, maintained_selection
+)
+from openpype.hosts.maya.api.pipeline import containerise
+
+
+def is_sequence(files):
```

Comment:
I wonder if this would be a good candidate to have in `openpype.lib`

Reply:
Think it might get lost and by that I mean only used for this and no one will use it other places. Think there was a conversation with @BigRoy about needing some unified approach to this.